src/Microsoft.ML.TimeSeries/AdaptiveSingularSpectrumSequenceModeler.cs

```diff
@@ -454,6 +454,7 @@ public AdaptiveSingularSpectrumSequenceModeler(IHostEnvironment env, ModelLoadCo
                 _wTrans = new CpuAlignedMatrixRow(_rank, _windowSize, SseUtils.CbAlign);
                 int i = 0;
                 _wTrans.CopyFrom(tempArray, ref i);
+                _y = new CpuAlignedVector(_rank, SseUtils.CbAlign);
```

Comment:
Where is this line coming from? Why is this change?

Reply:
I realized we also need to save this vector to disk if the matrix is saved because they go hand in hand. I have made this change.

---
In reply to: [236520664](https://github.com/dotnet/machinelearning/pull/1727#discussion_r236520664) [](ancestors = 236520664)